ext/stringio/stringio.c

```diff
@@ -1406,17 +1411,14 @@ strio_readline(int argc, VALUE *argv, VALUE self)
 
 /*
  * call-seq:
- *   strio.each(sep=$/, chomp: false) {|line| block }         -> strio
- *   strio.each(limit, chomp: false) {|line| block }          -> strio
- *   strio.each(sep, limit, chomp: false) {|line| block }     -> strio
- *   strio.each(...)                                          -> anEnumerator
- *
- *   strio.each_line(sep=$/, chomp: false) {|line| block }     -> strio
- *   strio.each_line(limit, chomp: false) {|line| block }      -> strio
- *   strio.each_line(sep, limit, chomp: false) {|line| block } -> strio
- *   strio.each_line(...)                                      -> anEnumerator
- *
- * See IO#each.
+ *   each_line(sep = $/, **line_opts) {|line| ... }   -> self
+ *   each_line(limit, **line_opts) {|line| ... }      -> self
+ *   each_line(sep, limit, **line_opts) {|line| ... } -> self
+ *
+ * Calls the block with each remaining line read from the stream;
+ * does nothing if already at end-of-file;
+ * returns +self+.
+ * See {Line IO}[https://docs.ruby-lang.org/en/master/io_streams_rdoc.html#label-Line+IO].
```

Comment:
I think we should mention that `each` is an alias of `each_line`.

Reply:
Added.